torch/csrc/jit/ir.h

```diff
@@ -1016,6 +1016,15 @@ friend struct Block;
     }
     return n;
   }
+  Node* createTupleIndex(Value * tup, Value * index) {
```

Comment:
This should be: `createTupleIndex(Value* tup, int64_t index)`. It is weird to have to create a constant node just to use it. The node in the graph should be a constant attribute not an input.